pkg/configuration/configuration.go

```diff
@@ -46,6 +45,26 @@ const (
 
 	// NotificationDeliveryServiceMailgun is the notification delivery service to use during production
 	NotificationDeliveryServiceMailgun = "mailgun"
+
+	varNotificationDeliveryService = "notification.delivery.service"
+
+	// varDurationBeforeNotificationDeletion specifies the duration before a notification will be deleted
+	varDurationBeforeNotificationDeletion = "duration.before.notification.deletion"
+
+	// varMailgunDomain specifies the host operator mailgun domain
+	varMailgunDomain = "mailgun.domain"
+
+	// varMailgunAPIKey specifies the host operator mailgun api key
+	varMailgunAPIKey = "mailgun.api.key"
+
+	// mailgunDomain defines mailgun domain key
+	mailgunDomain = "mailgun-domain"
+
+	// mailgunAPIKey defines mailgun api key
+	mailgunAPIKey = "mailgun-api-key"
+
+	// defaultNamesapce specifies  the default k8s namespace to use
+	defaultNamesapce = "toolchain-host-operator"
```

Comment:
Use k8sutil.GetWatchNamespace() instead

Reply:
hm, it looks like i get an error saying WATCH_NAMESPACE must be set. should i set it here and then use `k8util.GetWatchNamespace()`?

Reply:
It’s set in runtime: https://github.com/codeready-toolchain/host-operator/blob/ed9d063c090d644102211a08fc3820e819be4fdb/deploy/operator.yaml#L24
But you will need to set it in your unit tests.

Reply:
added here: bb686193a45963333e4a7b835b4039133d015688